pkg/interceptors/cel/cel.go

```diff
@@ -124,23 +124,12 @@ func makeEvalContext(body []byte, h http.Header, url string) (map[string]interfa
 }
 
 func (w *Interceptor) Process(ctx context.Context, r *triggersv1.InterceptorRequest) *triggersv1.InterceptorResponse {
-	b, err := json.Marshal(r.InterceptorParams)
-	if err != nil {
-		return &triggersv1.InterceptorResponse{
-			Continue: false,
-			Status:   status.Newf(codes.InvalidArgument, "failed to marshal json: %v", err),
-		}
+	p := triggersv1.CELInterceptor{}
+	if err := interceptors.UnmarshalParams(r.InterceptorParams, &p); err != nil {
+		return interceptors.Fail(status.Newf(codes.InvalidArgument, "failed to parse interceptor params: %v", err))
```

Comment:
We should wrap the error (here and elsewhere in this PR) 
```suggestion
		return interceptors.Fail(status.Newf(codes.InvalidArgument, "failed to parse interceptor params: %w", err))
```

Reply:
That fails govet with `pkg/interceptors/github/github.go:65:28: Newf call has error-wrapping directive %w`